jekyll/_cci2_ja/plan-free.adoc

```diff
@@ -34,4 +34,4 @@ Docker、 Linux、 Windows、および macOS のさまざまなリソースク
 
 === サポート
 
-https://discuss.circleci.com/[community forums]、 https://support.circleci.com/hc/ja[support portal]、および https://support.circleci.com/hc/ja/requests/new[global ticket-based system] からサポートをご利用いただけます。
\ No newline at end of file
+Scale プランでは、 https://discuss.circleci.com/[コミュニティフォーラム]、 https://support.circleci.com/hc/ja[サポートポータル]、および https://support.circleci.com/hc/ja/requests/new[グローバルチケットシステム] もご利用いただけます。
```

Comment:
```suggestion
Scale プランでは、 https://discuss.circleci.com/[コミュニティフォーラム]、 https://support.circleci.com/hc/ja[サポートポータル]、および https://support.circleci.com/hc/ja/requests/new[チケットベースのグローバル サポート] もご利用いただけます。
```

haha, i am not too sure which expression is best but the suggestion about is based on what we label for the same thing on our 🇯🇵  Pricing page:

For your consideration!
Please feel free to reject or update accordingly :)

<img width="782" alt="Screen Shot 2022-03-28 at 15 19 04" src="https://user-images.githubusercontent.com/2164346/160337699-2117aeec-6f98-4534-9ae1-0d5c2ab3a173.png">

Reply:
Thank you for pointing this out! I'll update other pages in next PR.